frontend/pages/index.tsx

```diff
@@ -22,34 +22,32 @@ function HomePage() {
         <title>{createPageTitle(t, 'home')}</title>
       </Head>
       <Cover />
-      <Typography variant="h4" color="secondary">Kort om oss</Typography>
+      <Typography variant="h4" color="secondary">{t('homePage:about')}</Typography>
       <Typography>
-        D-sektionen inom TLTH är en ideell organisation för
-        studenter och alumner vid programmen
+        {t('homePage:intro')}
         <Link
           href="https://www.lth.se/utbildning/datateknik300/"
           color="primary"
           rel="noopener noreferrer"
           target="_blank"
         >
           {' '}
-          Datateknik
+          {t('homePage:D')}
           {' '}
         </Link>
-        och
+        {t('common:and')}
         <Link
           href="https://www.lth.se/utbildning/informations-och-kommunikationsteknik/"
           color="secondary"
           rel="noopener noreferrer"
           target="_blank"
         >
           {' '}
-          InfoCom
-
+          {t('homePage:C')}
```

Comment:
Splitting translations up like this isn't ideal (if we consider other languages), but for now it's fine.